### PR TITLE
feat(devnet-docs): added docs for devnet'

### PR DESCRIPTION
**Pull request by BLuEScioN**
## Description

Describe the changes that were made in this pull request. When possible start with the motivations behind the change. Be sure to also include the following information:

1. Motivation for change?
We are releasing devnet for platform soon. These docs are documenting this new feature
3. What was changed?
Just new docs
5. Link to relevant issues?

## Checklist

- [ X] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ X] New links to files and images were verified
- [ X] For fixes/refactors: all existing references were identified and replaced
- [ X] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ X] Clear code samples were provided
- [ X] People were tagged for review

### CODE REVIEW

File: docs/platform/devnet.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+### Active Development Session
+
+When Devnet is ready, the status will change to 'Active', and a 30-minute session countdown will begin. This timer manages how long a devnet session will last. Once the 30 minutes are up the devent instance will automatically be shutdown.
+
+![Devnet Active](./images/devnet-active.jpeg)
+
+### Exploring the Devnet Dashboard
+
+With Devnet active, clicking on the status pill again will present the option to 'View Devnet Details'. Selecting this takes you to the Devnet dashboard, where you can interact with a block explorer and an "assembly line" of mined blocks.Transactions within each block are accessible here, and a search function is available for efficient block navigation.
```
Comment (ryanwaits) [inline docs/platform/devnet.md:46]:
```suggestion
With Devnet active, clicking on the status pill again will present the option to 'View Devnet Details'. Selecting this takes you to the Devnet dashboard, where you can interact with a block explorer and an "assembly line" of mined blocks. Transactions within each block are accessible here, and a search function is available for efficient block navigation.
```

Comment (ryanwaits) [reply docs/platform/devnet.md:46]: just a typo on spacing

**Comment by ryanwaits** [comment]
this lgtm on paper - however since the feature is not live i cant follow along with the guide to verify everything is accurate...

will wait for this feature to be merged and revisit 🙏

**Review by ryanwaits — CHANGES_REQUESTED** [review]
minor typo fix

**Comment by BLuEScioN** [comment]
> this lgtm on paper - however since the feature is not live i cant follow along with the guide to verify everything is accurate, is there a sandbox environment that i can be using?
> 
> otherwise will wait for this feature to be merged and revisit 🙏

We are trying to test on platform.dev.hiro.so, however there is a bug currently that is breaking devnet atm. I can let you know once that is fixed and you can inspect the new feature yourself. Probably tomorrow

**Comment by ryanwaits** [comment]
everything checks out nicely on `platform.dev.hiro.so` ✨

**Review by ryanwaits — APPROVED** [review]

**Comment by andresgalante** [comment]
We;'ll merge this when we release the feature

**Pull request merged**